<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisDialog</class>
 <widget class="QDialog" name="RegisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="Password">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="UsernameEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="PassEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="rightUser">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Username">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="IDEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="PassEdit2"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="Password2">
        <property name="text">
         <string>再次输入密码</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="rightPass">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="rightID">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="ID">
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>sss</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="regisRes.qrc">:/image/images(2).jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>270</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="confirmButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>UsernameEdit</tabstop>
  <tabstop>PassEdit</tabstop>
  <tabstop>PassEdit2</tabstop>
  <tabstop>IDEdit</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="regisRes.qrc"/>
  <include location="testqt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>UsernameEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>RegisDialog</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisDialog</receiver>
   <slot>slot3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisDialog</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>RegisDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PassEdit2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegisDialog</receiver>
   <slot>slot4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
  <slot>slot4()</slot>
 </slots>
</ui>
